CHANGELOG.txt

```diff
@@ -7,8 +7,8 @@ Changelog
  * Maintenance: Update BeautifulSoup upper bound to 4.12.x (scott-8)
 
 
-5.2 LTS (1.11.2023)
-~~~~~~~~~~~~~~~~~~~
+5.2 (xx.xx.xxxx) - IN DEVELOPMENT
```

Comment:
Sorry, looks like you've rolled this back too far now! The "LTS" _should_ be added here.

Reply:
Oh, I will correct that.

Reply:
Thanks - just the underline to fix now.

Reply:
My bad. Sorry, this has to take so much revisions. This is my first time working on a release note